client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -67,10 +68,12 @@ class PurchasePlanDetails extends Component {
 		} );
 
 		return (
-			<div>
```

Comment:
Alternatively, we can get something similar with something like this and no class or CSS needed:

```js
{ translate( 'Billed monthly' ) }
{ '  ' }
<Button
	onClick={ this.handleMonthlyToYearlyButtonClick }
	primary
	compact
>
```